style.css

```diff
@@ -130,90 +130,509 @@ header {
   opacity: 0.5;
 }
 
-@media only screen and (min-width: 768px) {
-  header {
-    position: absolute;
-    width: 1440px;
-    top: 1px;
-    background: #fff;
-  }
-
-  #logo {
-    width: 76px;
-    height: 20px;
-    margin-left: 150%;
-  }
-
-  .header {
-    position: relative;
-    width: 1440px;
-    height: 718px;
-    background-color: #fff;
-    background-image: url(images/bg/Header\ new\ bg.png);
-    background-repeat: no-repeat;
-    background-size: 1260px;
-    background-position-x: 70px;
-    border-bottom-left-radius: 60px;
-  }
-
-  .nav_links li a {
-    font-size: 15px;
-  }
-
-  .nav_links {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    gap: 25px;
-    margin-right: 10%;
-  }
-
-  .menu {
-    display: none;
-  }
-
-  /* .Header container */
-  .header_container {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 0;
-    gap: 12px;
-    position: absolute;
-    width: 920px;
-    height: 268px;
-    left: 260px;
-    top: calc(50% - 268px / 2);
-  }
-
-  .header_container h1 {
-    width: 420px;
-    height: 122px;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 48px;
-    line-height: 60px;
-    display: flex;
-    align-items: center;
-    letter-spacing: 0.37px;
-    color: #091e42;
-    flex: none;
-    order: 0;
-    flex-grow: 0;
-  }
-
-  .header_container p {
-    width: 920px;
-    height: 98px;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 20px;
-    line-height: 28px;
-    color: #344563;
-    flex: none;
-    order: 1;
-    flex-grow: 0;
-  }
+/* Cards works container */
+.cards_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  left: -1%;
+  top: 718px;
+}
+
+.card_works {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 542px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] The margin-left and margin-right on the individual cards are too wide. I would advise that you reduce them. Doing this will make your design look identical to that of the Figma template that you chose. Kindly see the different screenshots below for comparison 💻 🧑‍⚕️:

### Your Design

![Screenshot-1](https://user-images.githubusercontent.com/77106962/234291263-53a1923b-5cce-4f83-a5f0-f08fd0ee6386.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/234291543-e3678e0b-ef5e-46c2-a8cf-e92cb7dab72a.png)